iam/app.py

```diff
@@ -14,178 +14,110 @@
 
 import getpass
 import logging
-import secrets
-from datetime import datetime
 
 import click
 import firebase_admin
-from firebase_admin import auth, credentials
+from firebase_admin import credentials
 from flask import Flask, jsonify, request
 from flask_admin import Admin
 from flask_admin.contrib.sqla import ModelView
 from flask_basicauth import BasicAuth
 from flask_cors import CORS
 from flask_migrate import Migrate
-from jose import jwk, jwt
+from flask_restplus import Api
 from raven.contrib.flask import Sentry
 from sqlalchemy.orm.exc import NoResultFound
 
 from . import settings
 from .models import Organization, Project, User, db
 
 
-def create_app():
-    app = Flask(__name__)
-    app.config.from_object(settings.Settings)
+app = Flask(__name__)
+api = Api(
+    title="IAM API",
+    version="0.0.1",
+    description="Identity and access management",
+)
 
-    Migrate(app, db)
-    db.init_app(app)
 
-    if app.config['SENTRY_DSN']:
-        sentry = Sentry(dsn=app.config['SENTRY_DSN'], logging=True,
+def init_app(application, interface):
+    from . import resources
+
+    application.config.from_object(settings.Settings)
+
+    Migrate(application, db)
```

Comment:
Maybe make `db` an argument of the function, too? Since you're now using the local `application` and `interface` but global `db`.